src/worker.cc

```diff
@@ -701,8 +704,7 @@ void from_json(rapidjson::Document& doc, Options& options) {
   }
 
   // Option to use timestamps when computing elapsed time for matched routes
-  options.set_use_timestamps(
-      rapidjson::get_optional<bool>(doc, "/use_timestamps").get_value_or(false));
+  options.set_use_timestamps(rapidjson::get<bool>(doc, "/use_timestamps", false));
```

Comment:
you'll see a bunch of those: I made all the boolean options one-liners for purely cosmetic reasons. I also did check that none had a `has_xx` call somewhere which could've messed things up.